webapp/package-lock.json

```diff
@@ -1,18758 +1,8 @@
 {
 	"name": "focalboard",
 	"version": "0.9.1",
-	"lockfileVersion": 2,
+	"lockfileVersion": 1,
```

Comment:
Looks like you are accidentally rolling back to version 1 of npm (probably because you are not using the last version of node locally).

Reply:
Ugh.  Ok, I'll revert package-lock

Reply:
Reverted via https://github.com/mattermost/focalboard/pull/1369/commits/03daaf722a0bff201708cf27b0434c8fbfef4f4c